src/main/java/edu/cornell/kfs/pdp/service/impl/CuPendingTransactionServiceImpl.java

```diff
@@ -190,8 +193,8 @@ else if (paymentGroup.getDisbursementType().getCode().equals(PdpConstants.Disbur
                 glPendingTransaction.setChartOfAccountsCode(coaCode);
                 glPendingTransaction.setFinancialSubObjectCode(KFSConstants.getDashFinancialSubObjectCode());
                 
-                // KFSUPGRADE-943
-                glPendingTransaction.setSubAccountNumber(KFSConstants.getDashSubAccountNumber());              
+                // KFSPTS-12390
```

Comment:
this comment can be removed, we don't need this on EDU package classes for the most part